File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String answer = "";
        String a = "abcdefghij";
        String [] arr = String.valueOf(age).split("");
        for (int i = 0; i < arr.length; i++) {
            answer += a.charAt(Integer.valueOf(arr[i]));
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.py
def solution(age):
    answer = ''
    #우주행성 식 나이 연산 리턴
    #알파벳 담긴 리스트
    num = ['a','b','c','d','e','f','g','h','i','j']
    #나이를 str로 변환
    for i in str(age):
        answer += num[int(i)]
    return answer